<!DOCTYPE html>
<html lang="en">
<div id="mc" class=" article lit-style content pmc-wm slang-all page-box">
    <!--main-content-->
    <div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'">
        <div class="fm-sec half_rhythm no_top_margin">
            <!-- <div class="fm-flexbox">
                           <div class="fm-citation">
                              <div class="citation-default">
                                 <div class="part1"><span id="pmcmata">Hum Genomics.</span> 2006; 2(4): 203â€“211. </div>
                                 <div class="part2"><span class="fm-vol-iss-date">Published online 2006 Jan 1. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1186%2F1479-7364-2-4-203" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1186/1479-7364-2-4-203</a></span></div>
                              </div>
                           </div>
                           <div class="fm-ids">
                              <div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3525152</span></div>
                              <div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/16460646">16460646</a></div>
                           </div>
                        </div>
                        <h1 class="content-title">Genomic analysis of a heterogeneous Mendelian phenotype: multiple novel alleles for inherited hearing loss in the Palestinian population</h1> -->
            <div class="half_rhythm">
                <div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Walsh%20T%5BAuthor%5D" id="A1" class="affpopup" co-rid="_co_idm140672623460288" co-class="co-affbox">Tom Walsh</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rayan%20AA%5BAuthor%5D"
                        id="A2" class="affpopup" co-rid="_co_idm140672623458352" co-class="co-affbox">Amal Abu Rayan</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sa%27ed%20JA%5BAuthor%5D" id="A3" class="affpopup" co-rid="_co_idm140672623455696"
                        co-class="co-affbox">Judeh Abu Sa'ed</a>,<sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shahin%20H%5BAuthor%5D" id="A4" class="affpopup" co-rid="_co_idm140672621905568" co-class="co-affbox">Hashem Shahin</a>,<sup>2,</sup><sup>3</sup>                    <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shepshelovich%20J%5BAuthor%5D" id="A5" class="affpopup" co-rid="_co_idm140672621902112" co-class="co-affbox">Jeanne Shepshelovich</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lee%20MK%5BAuthor%5D"
                        id="A6" class="affpopup" co-rid="_co_idm140672621899456" co-class="co-affbox">Ming K Lee</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hirschberg%20K%5BAuthor%5D" id="A7" class="affpopup" co-rid="_co_idm140672620475312"
                        co-class="co-affbox">Koret Hirschberg</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tekin%20M%5BAuthor%5D" id="A8" class="affpopup" co-rid="_co_idm140672620473088" co-class="co-affbox">Mustafa Tekin</a>,<sup>5</sup>                    <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Salhab%20W%5BAuthor%5D" id="A9" class="affpopup" co-rid="_co_idm140672620470864" co-class="co-affbox">Wa'el Salhab</a>,<sup>6</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Avraham%20KB%5BAuthor%5D"
                        id="A10" class="affpopup" co-rid="_co_idm140672620468640" co-class="co-affbox">Karen B Avraham</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=King%20MC%5BAuthor%5D" id="A11" class="affpopup" co-rid="_co_idm140672620466416"
                        co-class="co-affbox">Mary-Claire King</a>,<sup><img src="https://www.ncbi.nlm.nih.gov//corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"/></sup><sup>1</sup> and <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kanaan%20M%5BAuthor%5D"
                        id="A12" class="affpopup" co-rid="_co_idm140672620463696" co-class="co-affbox">Moien Kanaan</a><sup>2</sup></div>
                <div style="display:none" class="contrib-group aff-tip">
                    <div id="_co_idm140672623460288">
                        <h3 class="no_margin">Tom Walsh</h3>
                        <p><sup>1</sup>Departments of Medicine and Genome Sciences, University of Washington, Seattle, WA, USA</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Walsh%20T%5BAuthor%5D">Tom Walsh</a></div>
                    </div>
                    <div id="_co_idm140672623458352">
                        <h3 class="no_margin">Amal Abu Rayan</h3>
                        <p><sup>2</sup>Department of Life Sciences, Bethlehem University, Bethlehem, Palestinian Authority</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Rayan%20AA%5BAuthor%5D">Amal Abu Rayan</a></div>
                    </div>
                    <div id="_co_idm140672623455696">
                        <h3 class="no_margin">Judeh Abu Sa'ed</h3>
                        <p><sup>2</sup>Department of Life Sciences, Bethlehem University, Bethlehem, Palestinian Authority</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sa%27ed%20JA%5BAuthor%5D">Judeh Abu Sa'ed</a></div>
                    </div>
                    <div id="_co_idm140672621905568">
                        <h3 class="no_margin">Hashem Shahin</h3>
                        <p><sup>2</sup>Department of Life Sciences, Bethlehem University, Bethlehem, Palestinian Authority</p>
                        <p><sup>3</sup>Department of Human Genetics and Molecular Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shahin%20H%5BAuthor%5D">Hashem Shahin</a></div>
                    </div>
                    <div id="_co_idm140672621902112">
                        <h3 class="no_margin">Jeanne Shepshelovich</h3>
                        <p><sup>4</sup>Department of Pathology, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shepshelovich%20J%5BAuthor%5D">Jeanne Shepshelovich</a></div>
                    </div>
                    <div id="_co_idm140672621899456">
                        <h3 class="no_margin">Ming K Lee</h3>
                        <p><sup>1</sup>Departments of Medicine and Genome Sciences, University of Washington, Seattle, WA, USA</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Lee%20MK%5BAuthor%5D">Ming K Lee</a></div>
                    </div>
                    <div id="_co_idm140672620475312">
                        <h3 class="no_margin">Koret Hirschberg</h3>
                        <p><sup>4</sup>Department of Pathology, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Hirschberg%20K%5BAuthor%5D">Koret Hirschberg</a></div>
                    </div>
                    <div id="_co_idm140672620473088">
                        <h3 class="no_margin">Mustafa Tekin</h3>
                        <p><sup>5</sup>Division of Paediatric Genetics, Ankara University School of Medicine, Dikimevi, Ankara, Turkey</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Tekin%20M%5BAuthor%5D">Mustafa Tekin</a></div>
                    </div>
                    <div id="_co_idm140672620470864">
                        <h3 class="no_margin">Wa'el Salhab</h3>
                        <p><sup>6</sup>Department of Otolaryngology, St. Joseph's Hospital, Jerusalem, Israel</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Salhab%20W%5BAuthor%5D">Wa'el Salhab</a></div>
                    </div>
                    <div id="_co_idm140672620468640">
                        <h3 class="no_margin">Karen B Avraham</h3>
                        <p><sup>3</sup>Department of Human Genetics and Molecular Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Avraham%20KB%5BAuthor%5D">Karen B Avraham</a></div>
                    </div>
                    <div id="_co_idm140672620466416">
                        <h3 class="no_margin">Mary-Claire King</h3>
                        <p><sup>1</sup>Departments of Medicine and Genome Sciences, University of Washington, Seattle, WA, USA</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=King%20MC%5BAuthor%5D">Mary-Claire King</a></div>
                    </div>
                    <div id="_co_idm140672620463696">
                        <h3 class="no_margin">Moien Kanaan</h3>
                        <p><sup>2</sup>Department of Life Sciences, Bethlehem University, Bethlehem, Palestinian Authority</p>
                        <div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kanaan%20M%5BAuthor%5D">Moien Kanaan</a></div>
                    </div>
                </div>
            </div>
            <div class="half_rhythm">
                <div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140672623526752_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140672623526752_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140672623526752_cpl">Copyright and License information</a>                    <a href="/pmc/about/disclaimer/" style="margin-left: 1em">Disclaimer</a></div>
                <div class="fm-authors-info hide half_rhythm" id="idm140672623526752_ai" style="display:none">
                    <div class="fm-affl" id="I1"><sup>1</sup>Departments of Medicine and Genome Sciences, University of Washington, Seattle, WA, USA</div>
                    <div class="fm-affl" id="I2"><sup>2</sup>Department of Life Sciences, Bethlehem University, Bethlehem, Palestinian Authority</div>
                    <div class="fm-affl" id="I3"><sup>3</sup>Department of Human Genetics and Molecular Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</div>
                    <div class="fm-affl" id="I4"><sup>4</sup>Department of Pathology, Sackler School of Medicine, Tel Aviv University, Tel Aviv, Israel</div>
                    <div class="fm-affl" id="I5"><sup>5</sup>Division of Paediatric Genetics, Ankara University School of Medicine, Dikimevi, Ankara, Turkey</div>
                    <div class="fm-affl" id="I6"><sup>6</sup>Department of Otolaryngology, St. Joseph's Hospital, Jerusalem, Israel</div>
                    <div><sup><img src="https://www.ncbi.nlm.nih.gov//corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"/></sup>Corresponding author.</div>
                    <div><span class="contrib-email" id="A11">Mary-Claire King: <a href="mailto:dev@null" data-email="ude.notgnihsaw.u@gnikcm" class="oemail">ude.notgnihsaw.u@gnikcm</a></span> </div>
                </div>
                <div class="fm-article-notes hide half_rhythm" id="idm140672623526752_an" style="display:none">
                    <div class="fm-pubdate half_rhythm">Received 2005 Jul 19; Accepted 2005 Jul 19.</div>
                </div>
                <div class="permissions half_rhythm hide" id="idm140672623526752_cpl" style="display:none">
                    <div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Â©2006 Henry Stewart Publications</div>
                </div>
            </div>
            <div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article." />
        </div>
        <div class="sec" />
        <div id="abstract-1" lang="en" class="tsec sec">
            <h2 class="head no_bottom_margin" id="abstract-1title">Abstract</h2>
            <!--article-meta-->
            <div>
                <p class="p p-first-last">
                    Recessively inherited phenotypes are frequent in the Palestinian population, as the result of a historical tradition of marriages within extended kindreds, particularly in isolated villages. In order to characterise the genetics of inherited hearing loss
                    in this population, we worked with West Bank schools for the deaf to identify children with prelingual, bilateral, severe to profound hearing loss not attributable to infection, trauma or other known environmental exposure. Of 156
                    families enrolled, hearing loss in 17 families (11 per cent) was due to mutations in <em>GJB2 </em>(connexin 26), a smaller fraction of <em>GJB2</em>-associated deafness than in other populations. In order to estimate
                    how many different genes might be responsible for hearing loss in this population, we evaluated ten families for linkage to all 36 known human autosomal deafness-related genes, fully sequencing hearing-related genes at any linked sites
                    in informative relatives. Four families harboured four novel alleles of <em>TMPRSS3 </em>(<em>988Î”A </em>= 352stop), otoancorin (<em>1067A </em>&gt;<em>T </em>= D356V) and pendrin (<em>716T </em>&gt; A = V239D and <em>1001G </em>&gt;
                    T = 346stop). In each family, all affected individuals were homozygous for the critical mutation. Each allele was specific to one or a few families in the cohort; none were widespread. Since epidemiological tests of association of
                    mutations with deafness were not feasible for such rare alleles, we used functional and bioinformatics approaches to evaluate their consequences. In six other families, hearing loss was not linked to any known gene, suggesting that
                    these families harbour novel genes responsible for this phenotype. We conclude that inherited hearing loss is highly heterogeneous in this population, with most extended families acting as genetic isolates in this context. We also
                    conclude that the same genes are responsible for hearing loss in this population as elsewhere, so that gene discovery in these families informs the genetics of hearing loss worldwide.</p>
            </div>
            <div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetics, genomics, inherited, deafness, hearing loss, mutation, Palestinian, TMPRSS3, pendrin, otoancorin</span></div>
        </div>
        <div id="sec-1" class="tsec sec">
            <h2 class="head no_bottom_margin" id="sec-1title">Introduction</h2>
            <p class="p p-first">Extended kindreds from highly endogamous communities are ideally suited for identifying genes responsible for clinically important phenotypes. The unique demographic history of the Middle East has led to many such communities. For more than
                5,000 years and continuing to the present, the eastern shores of the Mediterranean have seen immigration of people from a wide variety of cultures. Villages were often established by a few extended families and, despite their geographical
                proximity, remained demographically isolated. For centuries, marriages have been arranged within extended families in these villages, leading to high levels of consanguinity and consequently high frequencies of recessive traits [<a href="#B1"
                    rid="B1" class=" bibr popnode">1</a>,<a href="#B2" rid="B2" class=" bibr popnode">2</a>].</p>
            <p>The willingness of these kindreds to participate in research has been of enormous help to geneticists. In order to identify genes responsible for inherited hearing loss, we have worked with consanguineous kindreds from the Palestinian population.
                As expected, we have found recessive alleles responsible for inherited hearing loss in such families. We have also discovered that an extremely high level of both allelic and locus heterogeneity characterises inherited hearing loss in
                this population.</p>
            <p class="p p-last">Systematic population-wide genomic analysis may be the most effective way to identify all critical genes and alleles for heterogeneous phenotypes in communities with many small isolates. We illustrate such a strategy in this paper. In such
                populations, most alleles of clinical importance are likely to be rare, found in only one or a few affected families, thus effectively precluding case-control approaches to test associations of alleles with the trait. Functional and bioinformatics
                approaches can be applied to these situations, and we illustrate such strategies applied to hearing loss phenotypes. The most important consequence of this heterogeneity is that such populations may offer the opportunity to identify and
                characterise a large number of alleles and genes critical to common phenotypes. They may, therefore, be even more valuable than previously recognised.</p>
        </div>
        <div id="sec-2" class="tsec sec">
            <h2 class="head no_bottom_margin" id="sec-2title">Methods</h2>
            <div id="sec-3" class="sec sec-first">
                <h3 id="sec-3title">Ascertainment of subjects</h3>
                <p class="p p-first-last">Probands are children with prelingual bilateral hearing loss who attend one of the following schools for deaf children: Eftah School for the Deaf in Bethlehem; Al Amal School for the Deaf in Hebron; the Association for the Deaf in Nablus;
                    and the Princess Basma Rehabilitation Center for Deaf Children in Jerusalem. Together, these schools serve nearly all hearing-impaired Palestinian children living on the West Bank. Children were referred to one of us (M.K.) by their
                    teachers. Families were contacted through teachers and social workers. If the family expressed an interest, they were visited at home (by M.K. and A.A.R.) to discuss the project in detail, explaining that the first steps in participation
                    would be physical and audiological examinations to exclude hearing loss due to infection or trauma and to evaluate severity and laterality of the hearing loss. For families who agreed, informed consent was obtained from parents and
                    assent from older children. Controls comprised 100 Palestinian adults with normal hearing, ascertained when blood samples were drawn at West Bank clinics for preventive health services. Controls agreed that their DNA samples could
                    be used anonymously for this project. The project is approved by the Human Subjects Division of the University of Washington, by the Helsinki Committee of Tel Aviv University and by the Human Subjects Committee of Bethlehem University.
                    Single Project Assurances for Tel Aviv University and for Bethlehem University have been obtained from the US Office for Human Research Protection (OHRP). Consenting deaf adults and children, hearing adults and hearing children older
                    than the oldest age of onset of inherited deafness in their family were enrolled.</p>
            </div>
            <div id="sec-4" class="sec">
                <h3 id="sec-4title">Genotyping and sequencing</h3>
                <p class="p p-first">Microsatellite markers within and flanking deafness genes were selected from the Genome Browser or designed using the simple repeat track of the Genome Browser [<a href="#B3" rid="B3" class=" bibr popnode">3</a>]. DNA samples were genotyped
                    and lod scores calculated as previously described [<a href="#B4" rid="B4" class=" bibr popnode">4</a>].</p>
                <p>For sequencing candidate genes, primer pairs were designed to amplify and sequence the entire coding regions and splice sites of all isoforms of <em>GJB2 </em>(connexin 26),[<a href="#B4" rid="B4" class=" bibr popnode">4</a>]<em>TMPRSS3</em>,[
                    <a href="#B5" rid="B5" class=" bibr popnode">5</a>] pendrin[<a href="#B6" rid="B6" class=" bibr popnode">6</a>] and otoancorin [<a href="#B7" rid="B7" class=" bibr popnode">7</a>]. Sequencing was carried out using BigDye terminator
                    3.1 sequencing chemistry (Applied Biosystems) and was then visualised on ABI 3100 sequencers. Sequences were aligned and compared using SeqHelp [<a href="#B8" rid="B8" class=" bibr popnode">8</a>].</p>
                <p class="p p-last">Mutations described in the text were genotyped in all probands and controls by sequencing or by polymerase chain reaction (PCR) and restriction enzyme digest. Restriction enzyme digests were made with <em>Mbo </em>II for <em>TMPRSS3.988delA, Hph </em>I
                    for pendrin <em>1001G </em>&gt;<em>T </em>and <em>Fok </em>I for otoancorin <em>1067A </em>&gt;<em>T</em>. Pendrin <em>716T </em>&gt;<em>A </em>was screened by DNA sequencing in probands and controls. All probands and controls were
                    also screened by PCR and restriction enzyme digest for previously published Palestinian and Lebanese deafness alleles of otoferlin,[<a href="#B9" rid="B9" class=" bibr popnode">9</a>]<em>TECTA</em>,[<a href="#B10" rid="B10" class=" bibr popnode">10</a>]
                    otoancorin[
                    <a href="#B7" rid="B7" class=" bibr popnode">7</a>] and whirlin [<a href="#B11" rid="B11" class=" bibr popnode">11</a>]. Restriction enzyme digests were made with <em>Dde </em>I for otoferlin <em>2416T </em>&gt;<em>A </em>(<em>Y730X</em>),
                    <em>Pfi</em>MI for <em>TECTAIVS9</em>(+<em>1</em>)G &gt;<em>A</em>, <em>Cac</em>81 for otoancorin <em>IVS12</em>(+<em>2</em>)<em>T </em>&gt;<em>C </em>and <em>Bvb</em>CI for whirlin <em>2332C </em>&gt;<em>T </em>(<em>R778X</em>). The
                    Î²-satellite insertion in <em>TMPRSS3 </em>was analysed by PCR, as published [<a href="#B5" rid="B5" class=" bibr popnode">5</a>].</p>
            </div>
            <div id="sec-5" class="sec sec-last">
                <h3 id="sec-5title">Localisation of GFP chimeras</h3>
                <p class="p p-first-last">The pendrin V239D mutant was created by PCR mutagenesis using the QuickChange Site-Directed Mutagenesis Kit (Stratagene) and subcloned into the green fluorescent protein (GFP) expression vector pEYFP-C1 (Clontech). Cell cultures, expression
                    of chimeras and microscopy were carried out as previously described [<a href="#B12" rid="B12" class=" bibr popnode">12</a>].</p>
            </div>
        </div>
        <div id="sec-6" class="tsec sec">
            <h2 class="head no_bottom_margin" id="sec-6title">Results</h2>
            <p class="p p-first">The cohort presently includes 156 probands and their families. Probands were characterised as having prelingual, bilateral hearing loss that could not be attributed to infection or trauma. No vision problems were present. More subtle syndromic
                signs in some individuals were detected subsequently, as described below.</p>
            <p>Because mutations in the gap junction <em>GJB2 </em>(connexin 26) are responsible for a substantial fraction of recessive inherited hearing loss worldwide,[<a href="#B13" rid="B13" class=" bibr popnode">13</a>] we first sequenced both exons
                of this gene in genomic DNA from all probands [<a href="#B4" rid="B4" class=" bibr popnode">4</a>]. We also screened for the neighbouring deletion of <em>GJB6 </em>[<a href="#B14" rid="B14" class=" bibr popnode">14</a>]. Of the 156 families
                in the project, inherited hearing loss in 17 families (11 per cent) was due to mutations in <em>GJB2</em>. Five different mutant alleles of <em>GJB2 </em>were present [<a href="#B4" rid="B4" class=" bibr popnode">4</a>]. The fraction of
                inherited hearing loss due to <em>GJB2 </em>in this population is somewhat lower than in other populations.</p>
            <p>In order to assess genetic heterogeneity of inherited deafness in the population, we chose ten families, which each included at least four relatives with prelingual bilateral hearing loss who were wild-type for <em>GJB2</em>. (An average of
                six affected individuals per family were sampled.) All living relatives with hearing loss were evaluated by physical examination and audiology to distinguish inherited prelingual hearing loss from age-related hearing loss. Prelingual deafness
                was consistent with recessive inheritance in each family.</p>
            <p>We next carried out a hearing-loss-targeted linkage scan of the ten families, using 68 microsatellite markers within and flanking the 36 autosomal genes known to be responsible for inherited hearing loss (Table <a href="/pmc/articles/PMC3525152/table/T1/"
                    target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3em;">(Table1)</span></span><span>1</span></a>)
                [
                <a href="#B15" rid="B15" class=" bibr popnode">15</a>]. We tested for linkage to genes for either recessive or dominant hearing loss, because different alleles of the same gene may lead to hearing loss by different modes of inheritance.
                If markers were uninformative, other markers were substituted in that family [<a href="#B3" rid="B3" class=" bibr popnode">3</a>]. We noted whether affected individuals were homozygous for a linked haplotype, but did not exclude families
                with more than one linked haplotype that might reflect more than one deafness-associated allele.</p>
            <!--table ft1-->
            <!--table-wrap mode="anchored" t5-->
            <div class="table-wrap anchored whole_rhythm" id="T1">
                <h3>Table 1</h3>
                <!--caption a7-->
                <div class="caption">
                    <p>Genes responsible for inherited deafness (<em>DFN</em>) with neighbouring microsatellite markers used in the subgenome scan.</p>
                </div>
                <div class="xtable">
                    <table frame="hsides" rules="groups" class="rendered small default_table">
                        <thead>
                            <tr>
                                <th align="left" rowspan="1" colspan="1">Chromosome</th>
                                <th align="left" rowspan="1" colspan="1">Distance from pter (MB)</th>
                                <th align="left" rowspan="1" colspan="1">Gene</th>
                                <th align="left" rowspan="1" colspan="1">DFN locus</th>
                                <th align="left" rowspan="1" colspan="1">Size of gene (kb)</th>
                                <th align="left" colspan="3" rowspan="1">Flanking markers and kb spanned</th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">1 p36.31</td>
                                <td align="left" rowspan="1" colspan="1">6.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>ESPN</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B36</em></td>
                                <td align="left" rowspan="1" colspan="1">36</td>
                                <td align="left" rowspan="1" colspan="1">D1S253</td>
                                <td align="left" rowspan="1" colspan="1">D1S2731</td>
                                <td align="left" rowspan="1" colspan="1">342</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">1 p34.3</td>
                                <td align="left" rowspan="1" colspan="1">34.9</td>
                                <td align="left" rowspan="1" colspan="1"><em>GJB3</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A2</em></td>
                                <td align="left" rowspan="1" colspan="1">5</td>
                                <td align="left" rowspan="1" colspan="1">D 1S1570</td>
                                <td align="left" rowspan="1" colspan="1">D1S496</td>
                                <td align="left" rowspan="1" colspan="1">190</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">1 p34.2</td>
                                <td align="left" rowspan="1" colspan="1">40.9</td>
                                <td align="left" rowspan="1" colspan="1"><em>KCNQ4</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A2</em></td>
                                <td align="left" rowspan="1" colspan="1">55</td>
                                <td align="left" rowspan="1" colspan="1">D1S2706</td>
                                <td align="left" rowspan="1" colspan="1">D1S1188</td>
                                <td align="left" rowspan="1" colspan="1">273</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">2 p23.3</td>
                                <td align="left" rowspan="1" colspan="1">26.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>OTOF</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B9</em></td>
                                <td align="left" rowspan="1" colspan="1">102</td>
                                <td align="left" rowspan="1" colspan="1">D2S2350</td>
                                <td align="left" rowspan="1" colspan="1">D2S174</td>
                                <td align="left" rowspan="1" colspan="1">112</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">3 p21.31</td>
                                <td align="left" rowspan="1" colspan="1">46.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>TMIE</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B6</em></td>
                                <td align="left" rowspan="1" colspan="1">9</td>
                                <td align="left" rowspan="1" colspan="1">AC(46.635)</td>
                                <td align="left" rowspan="1" colspan="1">TG(46.779)</td>
                                <td align="left" rowspan="1" colspan="1">144</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">4 p16.1</td>
                                <td align="left" rowspan="1" colspan="1">6.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>WFS1</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A6/A14</em></td>
                                <td align="left" rowspan="1" colspan="1">33</td>
                                <td align="left" rowspan="1" colspan="1">TCCC(6.457)</td>
                                <td align="left" rowspan="1" colspan="1">D4S431</td>
                                <td align="left" rowspan="1" colspan="1">77</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">5 q31.3</td>
                                <td align="left" rowspan="1" colspan="1">140.9</td>
                                <td align="left" rowspan="1" colspan="1"><em>DIAPH1</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A1</em></td>
                                <td align="left" rowspan="1" colspan="1">104</td>
                                <td align="left" rowspan="1" colspan="1">D5S2119</td>
                                <td align="left" rowspan="1" colspan="1">D5S2010</td>
                                <td align="left" rowspan="1" colspan="1">254</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">5 q32</td>
                                <td align="left" rowspan="1" colspan="1">145.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>POU4F3</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A15</em></td>
                                <td align="left" rowspan="1" colspan="1">1</td>
                                <td align="left" rowspan="1" colspan="1">D 5S2099</td>
                                <td align="left" rowspan="1" colspan="1">D5S2033</td>
                                <td align="left" rowspan="1" colspan="1">628</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">6 p21.32</td>
                                <td align="left" rowspan="1" colspan="1">33.2</td>
                                <td align="left" rowspan="1" colspan="1"><em>COL11A2</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A13</em></td>
                                <td align="left" rowspan="1" colspan="1">30</td>
                                <td align="left" rowspan="1" colspan="1">D6S2414</td>
                                <td align="left" rowspan="1" colspan="1">D6S1701</td>
                                <td align="left" rowspan="1" colspan="1">449</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">6 q14.1</td>
                                <td align="left" rowspan="1" colspan="1">76.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>MY O6</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B37/A22</em></td>
                                <td align="left" rowspan="1" colspan="1">167</td>
                                <td align="left" rowspan="1" colspan="1">AC (76.528)</td>
                                <td align="left" rowspan="1" colspan="1">GT(76.674)</td>
                                <td align="left" rowspan="1" colspan="1">146</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">6 q23.2</td>
                                <td align="left" rowspan="1" colspan="1">133.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>EYA4</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A10</em></td>
                                <td align="left" rowspan="1" colspan="1">288</td>
                                <td align="left" rowspan="1" colspan="1">D6S975</td>
                                <td align="left" rowspan="1" colspan="1">D6S1722</td>
                                <td align="left" rowspan="1" colspan="1">517</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">7 p15.3</td>
                                <td align="left" rowspan="1" colspan="1">24.5</td>
                                <td align="left" rowspan="1" colspan="1"><em>DFN A5</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A5</em></td>
                                <td align="left" rowspan="1" colspan="1">51</td>
                                <td align="left" rowspan="1" colspan="1">D7S2444</td>
                                <td align="left" rowspan="1" colspan="1">D7S2493</td>
                                <td align="left" rowspan="1" colspan="1">477</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">7 q22.1</td>
                                <td align="left" rowspan="1" colspan="1">102.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>SLC26A5 </em>(Prestin)</td>
                                <td align="left" rowspan="1" colspan="1"><em>B61</em></td>
                                <td align="left" rowspan="1" colspan="1">93</td>
                                <td align="left" rowspan="1" colspan="1">D7S2509</td>
                                <td align="left" rowspan="1" colspan="1">D7S2504</td>
                                <td align="left" rowspan="1" colspan="1">478</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">7 q31.1</td>
                                <td align="left" rowspan="1" colspan="1">106.9</td>
                                <td align="left" rowspan="1" colspan="1"><em>SLC26A4 </em>(Pendrin)</td>
                                <td align="left" rowspan="1" colspan="1"><em>B4</em></td>
                                <td align="left" rowspan="1" colspan="1">57</td>
                                <td align="left" rowspan="1" colspan="1">D7S496</td>
                                <td align="left" rowspan="1" colspan="1">D7S2459</td>
                                <td align="left" rowspan="1" colspan="1">177</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">8 q22.3</td>
                                <td align="left" rowspan="1" colspan="1">102.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>TFCP2L3</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A28</em></td>
                                <td align="left" rowspan="1" colspan="1">177</td>
                                <td align="left" rowspan="1" colspan="1">D8S521</td>
                                <td align="left" rowspan="1" colspan="1">D8S1046</td>
                                <td align="left" rowspan="1" colspan="1">761</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">9 q21.13</td>
                                <td align="left" rowspan="1" colspan="1">72.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>TMC1</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B7/B11/A36</em></td>
                                <td align="left" rowspan="1" colspan="1">315</td>
                                <td align="left" rowspan="1" colspan="1">D9S1837</td>
                                <td align="left" rowspan="1" colspan="1">D9S1876</td>
                                <td align="left" rowspan="1" colspan="1">48</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">9 q32</td>
                                <td align="left" rowspan="1" colspan="1">114.3</td>
                                <td align="left" rowspan="1" colspan="1"><em>WHRN</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B31</em></td>
                                <td align="left" rowspan="1" colspan="1">103</td>
                                <td align="left" rowspan="1" colspan="1">D9S1824</td>
                                <td align="left" rowspan="1" colspan="1">D9S1855</td>
                                <td align="left" rowspan="1" colspan="1">619</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">10 p12.1</td>
                                <td align="left" rowspan="1" colspan="1">26.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>MY03A</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B30</em></td>
                                <td align="left" rowspan="1" colspan="1">278</td>
                                <td align="left" rowspan="1" colspan="1">D10S2481</td>
                                <td align="left" rowspan="1" colspan="1">D10S1775</td>
                                <td align="left" rowspan="1" colspan="1">60</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">10 q21.1</td>
                                <td align="left" rowspan="1" colspan="1">56.0</td>
                                <td align="left" rowspan="1" colspan="1"><em>PCDH15</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B23</em></td>
                                <td align="left" rowspan="1" colspan="1">980</td>
                                <td align="left" rowspan="1" colspan="1">D10S546</td>
                                <td align="left" rowspan="1" colspan="1">D10S1642</td>
                                <td align="left" rowspan="1" colspan="1">321</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">10 q22.1</td>
                                <td align="left" rowspan="1" colspan="1">73.0</td>
                                <td align="left" rowspan="1" colspan="1"><em>CDH23</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B12</em></td>
                                <td align="left" rowspan="1" colspan="1">420</td>
                                <td align="left" rowspan="1" colspan="1">D10S584</td>
                                <td align="left" rowspan="1" colspan="1">D10S1694</td>
                                <td align="left" rowspan="1" colspan="1">152</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">11 p15.1</td>
                                <td align="left" rowspan="1" colspan="1">17.5</td>
                                <td align="left" rowspan="1" colspan="1"><em>USH1C </em>(Harmonin)</td>
                                <td align="left" rowspan="1" colspan="1"><em>B18</em></td>
                                <td align="left" rowspan="1" colspan="1">51</td>
                                <td align="left" rowspan="1" colspan="1">D11S902</td>
                                <td align="left" rowspan="1" colspan="1">D11S4138</td>
                                <td align="left" rowspan="1" colspan="1">268</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">11 q13.5</td>
                                <td align="left" rowspan="1" colspan="1">76.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>MY07A</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B2/A11</em></td>
                                <td align="left" rowspan="1" colspan="1">87</td>
                                <td align="left" rowspan="1" colspan="1">D11S4179</td>
                                <td align="left" rowspan="1" colspan="1">D11S4186</td>
                                <td align="left" rowspan="1" colspan="1">573</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">11 q23.3</td>
                                <td align="left" rowspan="1" colspan="1">120.5</td>
                                <td align="left" rowspan="1" colspan="1"><em>TECTA</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B21/A8/A12</em></td>
                                <td align="left" rowspan="1" colspan="1">88</td>
                                <td align="left" rowspan="1" colspan="1">D11S4089</td>
                                <td align="left" rowspan="1" colspan="1">D11S4107</td>
                                <td align="left" rowspan="1" colspan="1">60</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">12 q13.3</td>
                                <td align="left" rowspan="1" colspan="1">55.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>MY O1A</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A48</em></td>
                                <td align="left" rowspan="1" colspan="1">22</td>
                                <td align="left" rowspan="1" colspan="1">D12S1644</td>
                                <td align="left" rowspan="1" colspan="1">D12S1691</td>
                                <td align="left" rowspan="1" colspan="1">1</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">13 q12.11</td>
                                <td align="left" rowspan="1" colspan="1">19.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>GJB2 </em>(Connexin26)</td>
                                <td align="left" rowspan="1" colspan="1"><em>B1/A3</em></td>
                                <td align="left" rowspan="1" colspan="1">5</td>
                                <td align="left" rowspan="1" colspan="1">D 13S1316</td>
                                <td align="left" rowspan="1" colspan="1">D13S175</td>
                                <td align="left" rowspan="1" colspan="1">167</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">13 q12.11</td>
                                <td align="left" rowspan="1" colspan="1">19.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>GJB6 </em>(Connexin31)</td>
                                <td align="left" rowspan="1" colspan="1"><em>B1/A3</em></td>
                                <td align="left" rowspan="1" colspan="1">9</td>
                                <td align="left" rowspan="1" colspan="1">D 13S1316</td>
                                <td align="left" rowspan="1" colspan="1">D13S175</td>
                                <td align="left" rowspan="1" colspan="1">167</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">14 q12</td>
                                <td align="left" rowspan="1" colspan="1">30.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>COCH</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A9</em></td>
                                <td align="left" rowspan="1" colspan="1">16</td>
                                <td align="left" rowspan="1" colspan="1">D14S1021</td>
                                <td align="left" rowspan="1" colspan="1">D14S54</td>
                                <td align="left" rowspan="1" colspan="1">256</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">15 q15.3</td>
                                <td align="left" rowspan="1" colspan="1">41.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>STRC</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B16</em></td>
                                <td align="left" rowspan="1" colspan="1">19</td>
                                <td align="left" rowspan="1" colspan="1">TG(41.690)</td>
                                <td align="left" rowspan="1" colspan="1">TG(41.707)</td>
                                <td align="left" rowspan="1" colspan="1">17</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">16 p12.2</td>
                                <td align="left" rowspan="1" colspan="1">21.2</td>
                                <td align="left" rowspan="1" colspan="1"><em>CRYM</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A40</em></td>
                                <td align="left" rowspan="1" colspan="1">20</td>
                                <td align="left" rowspan="1" colspan="1">D16S3045</td>
                                <td align="left" rowspan="1" colspan="1">TTA(21.603)</td>
                                <td align="left" rowspan="1" colspan="1">661</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">16 p12.2</td>
                                <td align="left" rowspan="1" colspan="1">21.6</td>
                                <td align="left" rowspan="1" colspan="1"><em>OTOA</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B22</em></td>
                                <td align="left" rowspan="1" colspan="1">82</td>
                                <td align="left" rowspan="1" colspan="1">D16S3045</td>
                                <td align="left" rowspan="1" colspan="1">TTA(21.603)</td>
                                <td align="left" rowspan="1" colspan="1">661</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">17 p11.2</td>
                                <td align="left" rowspan="1" colspan="1">18.0</td>
                                <td align="left" rowspan="1" colspan="1"><em>MY015A</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B3</em></td>
                                <td align="left" rowspan="1" colspan="1">71</td>
                                <td align="left" rowspan="1" colspan="1">AC (17.958)</td>
                                <td align="left" rowspan="1" colspan="1">AA T(18.013)</td>
                                <td align="left" rowspan="1" colspan="1">55</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">17 q25.3</td>
                                <td align="left" rowspan="1" colspan="1">77.1</td>
                                <td align="left" rowspan="1" colspan="1"><em>ACTG1</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A20/A26</em></td>
                                <td align="left" rowspan="1" colspan="1">3</td>
                                <td align="left" rowspan="1" colspan="1">G T(77.007)</td>
                                <td align="left" rowspan="1" colspan="1">CA (77.107)</td>
                                <td align="left" rowspan="1" colspan="1">100</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">19 q13.33</td>
                                <td align="left" rowspan="1" colspan="1">55.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>MYH14</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A4</em></td>
                                <td align="left" rowspan="1" colspan="1">107</td>
                                <td align="left" rowspan="1" colspan="1">D19S866</td>
                                <td align="left" rowspan="1" colspan="1">D19S904</td>
                                <td align="left" rowspan="1" colspan="1">22</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">21 q22.13</td>
                                <td align="left" rowspan="1" colspan="1">36.8</td>
                                <td align="left" rowspan="1" colspan="1"><em>CLDN14</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B29</em></td>
                                <td align="left" rowspan="1" colspan="1">20</td>
                                <td align="left" rowspan="1" colspan="1">D21S1252</td>
                                <td align="left" rowspan="1" colspan="1">D21S167</td>
                                <td align="left" rowspan="1" colspan="1">369</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">21 q22.3</td>
                                <td align="left" rowspan="1" colspan="1">42.7</td>
                                <td align="left" rowspan="1" colspan="1"><em>TMPRSS3</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B8/B10</em></td>
                                <td align="left" rowspan="1" colspan="1">25</td>
                                <td align="left" rowspan="1" colspan="1">D21S1225</td>
                                <td align="left" rowspan="1" colspan="1">GT(43.277)</td>
                                <td align="left" rowspan="1" colspan="1">740</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">22 q12.3</td>
                                <td align="left" rowspan="1" colspan="1">35.1</td>
                                <td align="left" rowspan="1" colspan="1"><em>MYH9</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>A1720</em></td>
                                <td align="left" rowspan="1" colspan="1">107</td>
                                <td align="left" rowspan="1" colspan="1">D22S1173</td>
                                <td align="left" rowspan="1" colspan="1">D22S283</td>
                                <td align="left" rowspan="1" colspan="1">115</td>
                            </tr>
                            <tr>
                                <td align="left" rowspan="1" colspan="1">22 q13.1</td>
                                <td align="left" rowspan="1" colspan="1">36.4</td>
                                <td align="left" rowspan="1" colspan="1"><em>TRIOBP</em></td>
                                <td align="left" rowspan="1" colspan="1"><em>B28</em></td>
                                <td align="left" rowspan="1" colspan="1">89</td>
                                <td align="left" rowspan="1" colspan="1">TG(36.464)</td>
                                <td align="left" rowspan="1" colspan="1">CA (36.494)</td>
                                <td align="left" rowspan="1" colspan="1">30</td>
                            </tr>
                        </tbody>
                    </table>
                </div>
                <div class="largeobj-link align_right" id="largeobj_idm140672620383120"><a target="object" rel="noopener" href="/pmc/articles/PMC3525152/table/T1/?report=objectonly">Open in a separate window</a></div>
                <div class="tblwrap-foot">
                    <p><em>DFNA </em>loci are <em>DFN </em>loci associated with dominant (A) hearing loss; <em>DFNB </em>loci are <em>DFN </em>loci associated with recessive (B) hearing loss.</p>
                </div>
            </div>
            <p class="p">In six families, hearing loss was not linked to any known hearing-related gene. The model for linkage was recessive fully penetrant hearing loss with no sporadics; marker allele frequencies were estimated from the families in the sample. By
                genotyping all flanking markers shown in Table <a href="/pmc/articles/PMC3525152/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1,</span></span><span>1</span></a>,
                linkage could be excluded for all known genes by negative lod scores; however, no one marker was consistently sufficiently informative to exclude linkage. Genes for hearing loss in these families will be identified by genome-wide linkage
                analysis and positional cloning. In four other families, hearing loss was linked to locales of known deafness genes: <em>TMPRSS3 </em>on chromosome 21q23, pendrin on 7q31 and otoancorin on 16p12 (Figure <a href="/pmc/articles/PMC3525152/figure/F1/"
                    target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1</span></span><span>1</span></a>).</p>
            <!--fig ft0-->
            <!--fig mode=article f1-->
            <div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1">
                <a href="/pmc/articles/PMC3525152/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">
                    <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1-->
                    <div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140672620033296"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 1479-7364-2-4-203-1.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 1479-7364-2-4-203-1.jpg"
                            src="https://www.ncbi.nlm.nih.gov//pmc/articles/PMC3525152/bin/1479-7364-2-4-203-1.jpg" /></div>
                    <div class="largeobj-link align_right" id="largeobj_idm140672620033296">
                        <a target="object" rel="noopener" href="/pmc/articles/PMC3525152/figure/F1/?report=objectonly">Open in a separate window</a></div>
                </a>
                <div class="icnblk_cntnt" id="lgnd_F1">
                    <div><a class="figpopup" href="/pmc/articles/PMC3525152/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div>
                    <!--caption a7-->
                    <div class="caption">
                        <p><strong>Novel mutations responsible for inherited deafness in four kindreds</strong>. In each pedigree, filled symbols represent individuals with severe, bilateral, prelingual hearing loss. The mutation found in each family is
                            illustrated by a sequence below the appropriate pedigree, with mutations indicated on the sequences by red arrows. (a) In Family W, hearing loss is linked to markers flanking the serine protease <em>TMPRSS3 </em>on chromosome
                            21q22.3. Sequence of <em>TMPRSS3 </em>in affected members of the family revealed frameshift <em>988Î”A </em>(357stop), which abrogates serine protease activity. As predicted by linkage data, all affected relatives in Family
                            W are homozygous for <em>TMPRSS3.988Î”A </em>and all unaffected relatives are heterozygous or wild-type. (b) In Family BR, hearing loss is linked to markers flanking otoancorin on chromosome 16p12.2. The sequence of otoancorin
                            in affected members of the family revealed missense mutation <em>1067A </em>&gt; <em>T </em>(D356V), a highly non-conservative change that is likely to disrupt the otoancorin transmembrane structure. All affected relatives
                            in Family BR are homozygous for otoancorin <em>1067A </em>&gt; <em>T </em>(D356V) and all unaffected relatives are heterozygous or wild-type. D356V is the first deafness-associated missense mutation reported in otoancorin.
                            (c) In Family Y, hearing loss is linked to markers flanking pendrin on chromosome 7q31.1. The sequence of pendrin in affected members of the family revealed missense mutation <em>716T </em>&gt; <em>A </em>(V239D), which leads
                            to cellular mislocalisation of pendrin protein (see Figure 2). All affected relatives in Family Y are homozygous for pendrin <em>716T </em>&gt; <em>A </em>(V239D) and all unaffected relatives are heterozygous or wild-type.
                            (d) In Family BF, hearing loss is also linked to markers flanking pendrin, but to different alleles than were linked to the phenotype in Family Y. The sequence of pendrin in affected members of Family BF revealed <em>1001G </em>&gt;
                            <em>T</em>. RT-PCR of pendrin from lymphoblasts of affected relatives of Family BF indicates that <em>1001G </em>&gt; <em>T </em>alters splicing, leading to insertion of 41 intronic base pairs into the pendrin message and a
                            premature stop at codon 346 (red box). By homology with other sulphate transporters, truncation at this site would abrogate sulphate transporter activity of the protein. All affected relatives in Family BF are homozygous for
                            pendrin <em>1001G </em>&gt; <em>T </em>and all unaffected relatives are heterozygous or wild-type.</p>
                    </div>
                </div>
            </div>
            <div id="sec-7" class="sec">
                <h3 id="sec-7title">TMPRSS3</h3>
                <p class="p p-first">Family W included 11 individuals with prelingual, bilateral, severe to profound hearing loss, with thresholds poorer than 85 decibels (dB) at all frequencies. All ten living affected individuals were homozygous at markers D21S1225 and
                    GT(42.677), which flank <em>TMPRSS3 </em>(Figure <a href="/pmc/articles/PMC3525152/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1A).</span></span><span>1A</span></a>).
                    The lod score for linkage of hearing loss to the D21S1225-GT(42.67) haplotype was 9.26. We sequenced <em>TMPRSS3 </em>in an affected child from Family W and identified <em>988delA</em>, a frameshift mutation in exon 10 leading to a
                    stop at codon 357. All relatives of Family W were genotyped for <em>988delA</em>: all affected individuals were homozygous for the mutation and all hearing individuals were heterozygous carriers or wild-type.</p>
                <p>TMPRSS3 is a transmembrane serine protease, with a trypsin-like protease domain extending from residues 217 - 444 and conserved active sites at histidine 257, aspartic acid 304 and serine 401 [<a href="#B5" rid="B5" class=" bibr popnode">5</a>].
                    Truncation at residue 357 would eliminate the active site at serine 401 and much of the domain, abrogating <em>TMPRSS3 </em>serine protease activity. A signalling pathway in the inner ear controlled by proteolytic cleavage includes
                    activation by TMPRSS3 of the epithelial sodium channel (ENaC) [<a href="#B16" rid="B16" class=" bibr popnode">16</a>]. Truncated TMPRSS3 of Family W could not activate ENaC.</p>
                <p class="p p-last">Disruptions of proteolysis are associated with a wide variety of genetic disorders,[<a href="#B17" rid="B17" class=" bibr popnode">17</a>] but TMPRSS3 was the first protease associated with hearing loss, demonstrated by mutations that
                    defined <em>DFNB8 </em>and <em>DFNB10 </em>[<a href="#B5" rid="B5" class=" bibr popnode">5</a>]. The <em>DFNB10 </em>mutation, found in a Palestinian family, was remarkable, consisting of the insertion of 18 complete Î²-satellite repeat
                    monomers, which are normally present in mobile tandem arrays of up to several hundred kilobases (kb) on the short arms of acrocentric chromosomes. The Î²-satellite mutation was not present in any probands in our series, nor in any hearing
                    controls in our series. In addition to all affected members of Family W, three other probands in our series were homozygous for 988delA = 357stop, one hearing control was also heterozygous for this mutation.</p>
            </div>
            <div id="sec-8" class="sec">
                <h3 id="sec-8title">Otoancorin</h3>
                <p class="p p-first">Family BR included six individuals with prelingual, bilateral, moderate to severe hearing loss. Five affected individuals were homozygous for markers D16S3045 and TTA(21.603), which flank otoancorin. The sixth affected individual was homozygous
                    at TTA(21.603) and heterozygous at D16S3045, possibly reflecting an ancestral recombination event (Figure <a href="/pmc/articles/PMC3525152/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1"
                        co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1B).</span></span><span>1B</span></a>). The lod score for linkage of hearing loss to
                    the D16S3045 -TTA(21.603) haplotype was 3.49. We sequenced otoancorin in a deaf individual and identified mis-sense mutation 1067A &gt; T, which leads to substitution of valine for aspartic acid at residue 356 (D356V). All six affected
                    individuals were homozygous for D356V and all hearing individuals were heterozygous for the mutation or wild-type.</p>
                <p>Otoancorin was shown to be associated with hearing loss as the result of a single mutation, a presumptive splice variant IVS12(+2)T &gt; C, identified in a Palestinian family [<a href="#B7" rid="B7" class=" bibr popnode">7</a>]. Otoancorin
                    IVS12(+2)T &gt; C did not appear among any probands in our series, nor among any Palestinian hearing controls. Similarly, the Family BR mutation D356V did not appear among any other probands in our series, but one control was a heterozygous
                    carrier of the allele.</p>
                <p class="p p-last">Otoancorin is located at the interface between the apical surface of the sensory epithelia and its overlying acellular gels, and is expressed only in the inner ear [<a href="#B7" rid="B7" class=" bibr popnode">7</a>]. It remains attached
                    to the apical plasma membrane of epithelial cells via seven transmembrane helices. Residue 356 lies between the first and second transmembrane helices and is perfectly conserved in all otoancorin homologues that can be predicted from
                    current genomic sequences (human, chimpanzee, dog, mouse, rat, chicken and <em>Xenopus</em>) [<a href="#B18" rid="B18" class=" bibr popnode">18</a>]. D356V is a highly non-conservative substitution at this site. D356V is predicted
                    by Tmpred[<a href="#B19" rid="B19" class=" bibr popnode">19</a>] to disrupt the transmembrane domain structure and by the program 'Sorting Intolerant From Tolerant'[<a href="#B20" rid="B20" class=" bibr popnode">20</a>] not to be a
                    tolerated substitution. Otoancorin D356V is likely to be the second deafness-associated mutation and the first deafness-associated mis-sense mutation in this gene.</p>
            </div>
            <div id="sec-9" class="sec sec-last">
                <h3 id="sec-9title">Pendrin (SLC26A4)</h3>
                <p class="p p-first">Family Y included four individuals with prelingual, bilateral, severe to profound hearing loss. All were homozygous at markers D7S496 and D7S2459, which flank pendrin (Figure <a href="/pmc/articles/PMC3525152/figure/F1/" target="figure"
                        class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1C).</span></span><span>1C</span></a>).
                    In Family Y, the lod score for linkage of hearing loss to the D7S496-D7S2459 haplotype was 2.11, close to the maximum lod score possible in this family. We sequenced pendrin in an affected child from family Y and identified <em>716T </em>&gt;<em>A </em>in
                    exon 6, which causes a substitution of aspartic acid for valine at amino acid 239 (V239D). All relatives were genotyped for this mutation: all four affected individuals were homozygous for V239D and all hearing individuals were either
                    heterozygous or wild-type.</p>
                <p>
                    Mutations in pendrin can cause either non-syndromic hearing loss or Pendred syndrome, [
                    <a href="#B6" rid="B6" class=" bibr popnode">6</a>] which, in addition to sensorineural hearing loss, includes thyroid enlargement (goitre) and temporal bone abnormalities of the inner ear, ranging from isolated enlargement of the
                    vestibular aqueduct (EVA) to Mondini dysplasia--a complex malformation of the cochlear spiral [<a href="#B21" rid="B21" class=" bibr popnode">21</a>]. Therefore, we evaluated the temporal bones of the affected children of Family Y
                    by computed tomography (CT) scan. As Figure <a href="/pmc/articles/PMC3525152/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3em;">Figure2A</span></span><span>2A</span></a>                    indicates, the vestibular aqueduct is enlarged. There was no clinical evidence of thyroid abnorm-alities and serum thyroid-stimulating hormone and thyroxine concentrations in the four affected individuals were normal.</p>
                <!--fig ft0-->
                <!--fig mode=article f1-->
                <div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2">
                    <a href="/pmc/articles/PMC3525152/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">
                        <!--fig/graphic|fig/alternatives/graphic mode="anchored" m1-->
                        <div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140672619983888"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is 1479-7364-2-4-203-2.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is 1479-7364-2-4-203-2.jpg"
                                src="https://www.ncbi.nlm.nih.gov//pmc/articles/PMC3525152/bin/1479-7364-2-4-203-2.jpg" /></div>
                        <div class="largeobj-link align_right" id="largeobj_idm140672619983888">
                            <a target="object" rel="noopener" href="/pmc/articles/PMC3525152/figure/F2/?report=objectonly">Open in a separate window</a></div>
                    </a>
                    <div class="icnblk_cntnt" id="lgnd_F2">
                        <div><a class="figpopup" href="/pmc/articles/PMC3525152/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div>
                        <!--caption a7-->
                        <div class="caption">
                            <p><strong>Clinical, biological and evolutionary analyses of pendrin <em>716T </em>&gt; <em>A </em>(V239D) from Family Y</strong>. (a) CT scan of the temporal bones of a child from Family Y who is homozygous for <em>716T </em>&gt;
                                <em>A </em>(V239D) reveals enlargement of the vestibular aqueduct, or EVA (red arrows). EVA is characteristic of Pendred syndrome. All four homozygous relatives in Family Y have prelingual, severe hearing loss, with thresholds
                                &gt; 85 decibels at all frequencies. There was no clinical indication of goitre and serum thyroid-stimulating hormone and thyroxine levels were normal. (b) Intracellular localisation and trafficking of YFP-tagged mutant
                                (V239D) and wild-type (PDS) pendrin, prepared as previously described [<a href="#B12" rid="B12" class=" bibr popnode">12</a>,<a href="#B23" rid="B23" class=" bibr popnode">23</a>]. Living COS7 cells were transiently transfected
                                with YFP-V239D mutant pendrin (green) and with CFP-GPI (glycosylphosphatidylinositol [GPI] tagged with crimson fluorescent protein [CFP], red), which localises to the Golgi apparatus and the plasma membrane [<a href="#B12"
                                    rid="B12" class=" bibr popnode">12</a>]. After incubation, cells were visualised by confocal microscopy [<a href="#B12" rid="B12" class=" bibr popnode">12</a>]. Merged confocal images (yellow) indicate that pendrin V239D
                                is retained in the endoplasmic reticulum, whereas wild-type pendrin (PDS) colocalises with GPI to the Golgi apparatus and plasma membrane. (c) The predicted severity of V239D in Family Y was compared with other disease-associated
                                mis-sense mutations in pendrin (red diamonds) and to wild-type sequences of other human SLC26A anion transporters at homologous sites (blue symbols) using Grantham's amino acid difference formula [<a href="#B29" rid="B29"
                                    class=" bibr popnode">29</a>]. Some, but not all, pendrin mis-sense mutations are more diverged from wild-type pendrin than are other SLC26A transporters. By this measure, V239D is among the most divergent mis-sense mutations
                                of pendrin, particularly given the conservation of other SLC26A proteins at this site. (d) Palestinian and Turkish individuals with the <em>716A </em>(239D) mutation share an extended haplotype of 126 -260 kilobases flanking
                                pendrin (yellow boxes), based on markers polymorphic among wild-type (<em>716T</em>) haplotypes. Pendrin <em>716T </em>&gt; <em>A </em>has not been observed in any other Palestinian families with inherited hearing loss
                                or Turkish individuals with Pendred syndrome,[<a href="#B30" rid="B30" class=" bibr popnode">30</a>] suggesting that these two families may share a recent common ancestor.</p>
                        </div>
                    </div>
                </div>
                <p>Pendrin is a chloride -iodide transporter; mutations associated with Pendred syndrome can cause its retention in the endoplasmic reticulum [<a href="#B22" rid="B22" class=" bibr popnode">22</a>]. In order to test the consequences of the
                    V239D mis-sense mutation on intracellular localisation of pendrin, we evaluated trafficking of wild-type pendrin and of the V239D mutant protein in living cells using GFP chimeras [<a href="#B22" rid="B22" class=" bibr popnode">22</a>,
                    <a href="#B23" rid="B23" class=" bibr popnode">23</a>]. As shown in Figure
                    <a href="/pmc/articles/PMC3525152/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3em;">Figure2B,</span></span><span>2B</span></a>,
                    the V239D mutant pendrin is retained in the endoplasmic reticulum, whereas the wild-type protein targets the plasma membrane. Retention of pendrin in the endoplasmic reticulum is a major mechanism for Pendred syndrome,[<a href="#B12"
                        rid="B12" class=" bibr popnode">12</a>] suggesting that V239D is probably the pathogenic mutation in Family Y.</p>
                <p>We also developed a method for evaluating the severity of mis-sense mutations in pendrin, based on conservation across the large SLC26A family of solute carrier proteins and the growing number of known disease-associated pendrin missense
                    mutations [<a href="#B6" rid="B6" class=" bibr popnode">6</a>,<a href="#B24" rid="B24" class=" bibr popnode">24</a>-<a href="#B28" rid="B28" class=" bibr popnode">28</a>]. We compared the severity of mis-sense changes in pendrin with
                    naturally occurring variation across all proteins in the family. Figure <a href="/pmc/articles/PMC3525152/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3em;">Figure2C</span></span><span>2C</span></a>                    illustrates these comparisons as a 'diversity plot', calibrated against wild-type pendrin. For each amino acid site at which a pendrin mis-sense mutation has been reported, we plotted the amino acid difference based on Grantham's formula[
                    <a href="#B29" rid="B29" class=" bibr popnode">29</a>] for all naturally occurring amino acid differences in any SLC26A transporter. Then we compared scores of pendrin mis-senses to scores of naturally occurring variants. The mis-sense
                    mutation V239D is a less conservative substitution than any naturally occurring variant at the homologous site, and is among the least conservative changes observed at any site in the <em>SLC26A </em>gene family.</p>
                <p>Pendrin V239D did not occur in other Palestinian probands with hearing loss, nor among Palestinian controls; however, pendrin <em>716T </em>&gt;<em>A </em>(V239D) appeared in a Turkish brother and sister with prelingual hearing loss, hypothyroid
                    and EVA [<a href="#B30" rid="B30" class=" bibr popnode">30</a>]. The Palestinian and Turkish haplotypes harbouring <em>716A </em>&gt;<em>T </em>were identical for microsatellite markers spanning 262 kb, whereas other haplotypes in
                    this region in these families were diverse (Figure <a href="/pmc/articles/PMC3525152/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure2D).</span></span><span>2D</span></a>).
                    The mutation may have a common ancestor, although the families are not aware that they are related. The difference between normal thyroid function in the affected individuals in Family Y versus hypothyroidism in the affected individuals
                    in the Turkish family is not surprising, given the intra-familial variability of Pendred syndrome described for other mis-sense mutations [<a href="#B26" rid="B26" class=" bibr popnode">26</a>,<a href="#B31" rid="B31" class=" bibr popnode">31</a>].</p>
                <p>In Family BF, the lod score for linkage of hearing loss to the D7S496-D7S2459 haplotype was 1.66, which was not statistically significant but was close to the maximum possible for this family. Sequencing genomic DNA from Family BF revealed
                    a different mutation in pendrin. Four children with prelingual, bilateral, severe to profound hearing loss were homozygous at markers D7S496 and D7S2459 but for different alleles than in Family Y (Figure <a href="/pmc/articles/PMC3525152/figure/F1/"
                        target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">â€‹<span class="figpopup-sensitive-area" style="left: -3.5em;">(Figure1D).</span></span><span>1D</span></a>).
                    We sequenced pendrin in an affected child from Family BF and identified <em>1001G </em>&gt;<em>T</em>, a splice mutation in the last base pair of exon 8. All affected individuals in Family BF were homozygous for <em>1001G </em>&gt;<em>T </em>and
                    all hearing individuals were either heterozygous for the mutation or wild-type.</p>
                <p class="p p-last">Reverse transcriptase-PCR (RT-PCR) of the pendrin message from lymphoblast cDNA of a homozygous child revealed aberrant splicing, leading to insertion of 41 bp of intron 8 into the pendrin message and a premature stop at codon 346. The
                    region of pendrin homologous to other sulphate transporters includes the site of the truncation, suggesting that this mutation would abrogate sulphate transporter activity. The Family BF mutation <em>1001G </em>&gt;<em>T </em>has not
                    been previously reported. Mutation of the adjacent base pair, <em>1001</em>(+<em>1</em>)<em>G </em>&gt;<em>A</em>, however, which also leads to 346stop, is a founder mutation among English families with Pendred syndrome, probably including
                    those families in whom the syndrome was originally described [<a href="#B24" rid="B24" class=" bibr popnode">24</a>,<a href="#B32" rid="B32" class=" bibr popnode">32</a>].</p>
            </div>
        </div>
        <div id="sec-10" class="tsec sec">
            <h2 class="head no_bottom_margin" id="sec-10title">Discussion</h2>
            <p class="p p-first">Inherited hearing loss in Middle Eastern populations is highly heterogeneous, both in the number of genes involved and in the number of alleles at each gene. Of the ten families screened for linkage of deafness to all genes known to influence
                hearing loss, none carried any of the more than 100 previously identified alleles. All families with hearing loss who were wild-type for <em>GJB2 </em>harboured novel alleles, either in known genes (four families) or in genes yet to be
                identified (six families). Alleles responsible for hearing loss in this population are individually very rare.</p>
            <p>Allelic heterogeneity of recessive diseases in the Palestinian population is well documented [<a href="#B2" rid="B2" class=" bibr popnode">2</a>]. As pointed out by Zlotogora,[<a href="#B2" rid="B2" class=" bibr popnode">2</a>] the many unique
                mutations among patients with autosomal recessive diseases in this population reflect the high rate of <em>de novo </em>deleterious mutations occurring in all human genomes [<a href="#B33" rid="B33" class=" bibr popnode">33</a>]. <em>De novo </em>recessive
                alleles are generally not detectable because the associated phenotypes are not expressed in outbred populations. The appearance of multiple novel alleles of known deafness-related genes also suggests that hearing loss in this population
                is genetically similar to hearing loss worldwide, and that the phenotype is simply more common in this population because of consanguinity.</p>
            <p>More than 50 years before the studies of Mendel, Darwin or Galton, Joseph Adams recognised the importance of isolated communities to the study of human disease, noting that 'endemic peculiarities may be found in certain sequestered districts'
                [
                <a href="#B34" rid="B34" class=" bibr popnode">34</a>]. A very large number of the discoveries of modern human genetics have been as the result of studies of such 'sequestered' populations [<a href="#B35" rid="B35" class=" bibr popnode">35</a>].
                Historical demography of isolated populations may lead to the occurrence of population-specific alleles of relatively high frequency, such as the ancient mutations responsible for diseases found in the Ashkenazi Jewish population,[<a href="#B36"
                    rid="B36" class=" bibr popnode">36</a>,<a href="#B37" rid="B37" class=" bibr popnode">37</a>] and also many population-specific alleles that are individually rare, each present in only one a few extended families, as described here.</p>
            <p>Given the availability of genomics tools, it is now possible to develop gene discovery strategies best suited to the demography of each community. For phenotypes with a high level of allele and locus heterogeneity, identified in extended consanguineous
                families, we developed the following strategy. (1) For any gene with relatively widespread mutations (eg <em>GJB2</em>), we sequenced all affected individuals in all families. It was important to fully sequence such genes, rather than
                only screening known mutations, because both old and new alleles are likely to appear [<a href="#B4" rid="B4" class=" bibr popnode">4</a>]. It was also important to sequence all affected individuals, not only probands, because more than
                one gene for hearing loss might segregate in a family. (2) For families which are wild-type at the commonly mutant genes, we tested for linkage of the phenotype to all known disease genes by a 'subgenome scan' consisting of markers intragenic
                or immediately flanking all genes known to be associated with the phenotype. Because there are a large number of known genes for hearing loss, we first genotyped two such markers at each site, then added additional markers as necessary.
                (3) In each potentially linked family, we fully sequenced the known gene linked to the phenotype in an affected relative and an obligate carrier. Clearly, it was not sufficient to genotype only known disease alleles because it is likely
                that new families will harbour new alleles. Also, although in this project each novel allele was homozygous in affected individuals, it is certainly possible that deaf individuals in some families will be compound heterozygotes. (4) We
                screened each mutation in all probands and in a small series (100) of Palestinian controls to determine allele frequencies, but mutations were individually too rare for casecontrol comparisons to have adequate statistical power for epidemiological
                analyses. Therefore, to evaluate the consequence of mis-sense mutations, we developed and used both functional and bioinformatics approaches.</p>
            <p class="p p-last">In conclusion, extended consanguineous kindreds prove extremely valuable for the identification of novel genes for otherwise intractable, highly heterogeneous phenotypes. Often extended kindreds in our series evaluated thus far, four families
                carried novel mutations in known genes and six families are likely to harbour novel mutations in novel genes for hearing loss. The rate of discovery of new genes related to any phenotype will eventually reach an asymptote. The ratio of
                mutations in known versus unknown genes in the present study (4:6), however, suggests that in this cohort of 156 families, a large number of novel genes for hearing loss still await discovery.</p>
        </div>
        <div id="sec-11" class="tsec bk-sec">
            <h2 class="head no_bottom_margin" id="sec-11title">Acknowledgements</h2>
            <!--/article/back/sec/-->
            <p class="p p-first-last">We thank the families and the staff of the Palestinian schools for children with hearing loss for their enthusiastic participation. This project was supported by NIH R01 DC005641.</p>
        </div>
        <div id="ref-list-1" class="tsec sec">
            <h2 class="head no_bottom_margin" id="ref-list-1title">References</h2>
            <div class="ref-list-sec sec" id="reference-list">
                <ul class="back-ref-list" style="list-style-type:decimal;">
                    <li id="B1"><span class="mixed-citation">Jaber L, Halpern GJ, Shohat T. Trends in the frequencies of consanguineous marriages in the Israeli Arab community. <span><span class="ref-journal">Clin Genet. </span>2000;<span class="ref-vol">58</span>:106â€“110.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/11005142" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Trends+in+the+frequencies+of+consanguineous+marriages+in+the+Israeli+Arab+community&amp;author=L+Jaber&amp;author=GJ+Halpern&amp;author=T+Shohat&amp;volume=58&amp;publication_year=2000&amp;pages=106-110&amp;pmid=11005142&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B2"><span class="mixed-citation">Zlotogora J. Molecular basis of autosomal recessive diseases among the Palestinian Arabs. <span class="ref-journal">Am J Med Genet.</span> pp. 176â€“182. [<a href="https://pubmed.ncbi.nlm.nih.gov/11977175"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span>
                    </li>
                    <li id="B3"><span class="mixed-citation">Kent WJ, Sugnet CW, Furey TS. et al. The Human Genome Browser at UCSC. <span><span class="ref-journal">Genome Res. </span>2002;<span class="ref-vol">12</span>:996â€“1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC186604/">PMC free article</a>]</span>                        [<a href="https://pubmed.ncbi.nlm.nih.gov/12045153" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=The+Human+Genome+Browser+at+UCSC&amp;author=WJ+Kent&amp;author=CW+Sugnet&amp;author=TS+Furey&amp;volume=12&amp;publication_year=2002&amp;pages=996-1006&amp;pmid=12045153&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B4"><span class="mixed-citation">Walsh T, Walsh V, Vreugde S. et al. From flies' eyes to our ears: Mutations in a human class III myosin cause progressive nonsyndromic hearing loss DFNB30. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;
                        <span class="ref-vol">99</span>:7518â€“7523. doi: 10.1073/pnas.102091699.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC124268/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12032315"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [
                        <a href="//doi.org/10.1073%2Fpnas.102091699" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=From+flies'+eyes+to+our+ears:+Mutations+in+a+human+class+III+myosin+cause+progressive+nonsyndromic+hearing+loss+DFNB30&amp;author=T+Walsh&amp;author=V+Walsh&amp;author=S+Vreugde&amp;volume=99&amp;publication_year=2002&amp;pages=7518-7523&amp;pmid=12032315&amp;doi=10.1073/pnas.102091699&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B5"><span class="mixed-citation">Scott HS, Kudoh J, Wattenhofer M. et al. Insertion of beta-satellite repeats identifies a transmembrane protease causing both congenital and childhood onset autosomal recessive deafness. <span><span class="ref-journal">Nat Genet. </span>2001;
                        <span class="ref-vol">27</span>:59â€“63.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11137999" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Insertion+of+beta-satellite+repeats+identifies+a+transmembrane+protease+causing+both+congenital+and+childhood+onset+autosomal+recessive+deafness&amp;author=HS+Scott&amp;author=J+Kudoh&amp;author=M+Wattenhofer&amp;volume=27&amp;publication_year=2001&amp;pages=59-63&amp;pmid=11137999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B6"><span class="mixed-citation">Everett LA, Glaser B, Beck JC. et al. Pendred syndrome is caused by mutations in a putative sulphate transporter gene (PDS) <span><span class="ref-journal">Nat Genet. </span>1997;<span class="ref-vol">17</span>:411â€“422.
                        doi: 10.1038/ng1297-411.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9398842" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1038%2Fng1297-411" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Pendred+syndrome+is+caused+by+mutations+in+a+putative+sulphate+transporter+gene+(PDS)&amp;author=LA+Everett&amp;author=B+Glaser&amp;author=JC+Beck&amp;volume=17&amp;publication_year=1997&amp;pages=411-422&amp;pmid=9398842&amp;doi=10.1038/ng1297-411&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B7"><span class="mixed-citation">Zwaenepoel I, Mustapha M, Leibovici M. et al. Otoancorin, an inner ear protein restricted to the interface between the apical surface of sensory epithelia and their overlying acellular gels, is defective in autosomal recessive deafness DFNB22. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2002;
                        <span class="ref-vol">99</span>:6240â€“6245. doi: 10.1073/pnas.082515999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC122933/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11972037"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [
                        <a href="//doi.org/10.1073%2Fpnas.082515999" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Otoancorin,+an+inner+ear+protein+restricted+to+the+interface+between+the+apical+surface+of+sensory+epithelia+and+their+overlying+acellular+gels,+is+defective+in+autosomal+recessive+deafness+DFNB22&amp;author=I+Zwaenepoel&amp;author=M+Mustapha&amp;author=M+Leibovici&amp;volume=99&amp;publication_year=2002&amp;pages=6240-6245&amp;pmid=11972037&amp;doi=10.1073/pnas.082515999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B8"><span class="mixed-citation">Lee MK, Lynch ED, King MC. SeqHelp: A program to analyze molecular sequences utilizing common computational resources. <span><span class="ref-journal">Genome Res. </span>1998;<span class="ref-vol">8</span>:306â€“312.</span>
                        <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC310699/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9521933" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=SeqHelp:+A+program+to+analyze+molecular+sequences+utilizing+common+computational+resources&amp;author=MK+Lee&amp;author=ED+Lynch&amp;author=MC+King&amp;volume=8&amp;publication_year=1998&amp;pages=306-312&amp;pmid=9521933&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B9"><span class="mixed-citation">Yasunaga S, Grati M, Cohen-Salmon M. et al. A mutation in OTOF, encoding otoferlin, a FER-1-like protein, causes DFNB9, a nonsyndromic form of deafness. <span><span class="ref-journal">Nat Genet. </span>1999;
                        <span class="ref-vol">21</span>:363â€“369. doi: 10.1038/7693.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10192385" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1038%2F7693" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+mutation+in+OTOF,+encoding+otoferlin,+a+FER-1-like+protein,+causes+DFNB9,+a+nonsyndromic+form+of+deafness&amp;author=S+Yasunaga&amp;author=M+Grati&amp;author=M+Cohen-Salmon&amp;volume=21&amp;publication_year=1999&amp;pages=363-369&amp;pmid=10192385&amp;doi=10.1038/7693&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B10"><span class="mixed-citation">Mustapha M, Weil D, Chardenoux S. et al. An alpha-tectorin gene defect causes a newly identified autosomal recessive form of sensorineural pre-lingual non-syndromic deafness, DFNB21. <span><span class="ref-journal">Hum Mol Genet. </span>1999;
                        <span class="ref-vol">8</span>:409â€“412. doi: 10.1093/hmg/8.3.409.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9949200" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1093%2Fhmg%2F8.3.409" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=An+alpha-tectorin+gene+defect+causes+a+newly+identified+autosomal+recessive+form+of+sensorineural+pre-lingual+non-syndromic+deafness,+DFNB21&amp;author=M+Mustapha&amp;author=D+Weil&amp;author=S+Chardenoux&amp;volume=8&amp;publication_year=1999&amp;pages=409-412&amp;pmid=9949200&amp;doi=10.1093/hmg/8.3.409&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B11"><span class="mixed-citation">Mburu P, Mustapha M, Varela A. et al. Defects in whirlin, a PDZ domain molecule involved in stereocilia elongation, cause deafness in the whirler mouse and families with DFNB31. <span><span class="ref-journal">Nat Genet. </span>2003;
                        <span class="ref-vol">34</span>:421â€“428. doi: 10.1038/ng1208.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12833159" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1038%2Fng1208" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Defects+in+whirlin,+a+PDZ+domain+molecule+involved+in+stereocilia+elongation,+cause+deafness+in+the+whirler+mouse+and+families+with+DFNB31&amp;author=P+Mburu&amp;author=M+Mustapha&amp;author=A+Varela&amp;volume=34&amp;publication_year=2003&amp;pages=421-428&amp;pmid=12833159&amp;doi=10.1038/ng1208&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B12"><span class="mixed-citation">Rotman-Pikielny P, Hirschberg K, Maruvada P. et al. Retention of pendrin in the endoplasmic reticulum is a major mechanism for Pendred syndrome. <span><span class="ref-journal">Hum Mol Genet. </span>2002;
                        <span class="ref-vol">11</span>:2625â€“2633. doi: 10.1093/hmg/11.21.2625.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12354788" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1093%2Fhmg%2F11.21.2625" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Retention+of+pendrin+in+the+endoplasmic+reticulum+is+a+major+mechanism+for+Pendred+syndrome&amp;author=P+Rotman-Pikielny&amp;author=K+Hirschberg&amp;author=P+Maruvada&amp;volume=11&amp;publication_year=2002&amp;pages=2625-2633&amp;pmid=12354788&amp;doi=10.1093/hmg/11.21.2625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B13"><span class="mixed-citation">Nance WE, Kearsey MJ. Relevance of connexin deafness (DFNB1) to human evolution. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">74</span>:1081â€“1087. doi: 10.1086/420979.</span>
                        <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182073/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15079193" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1086%2F420979" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Relevance+of+connexin+deafness+(DFNB1)+to+human+evolution&amp;author=WE+Nance&amp;author=MJ+Kearsey&amp;volume=74&amp;publication_year=2004&amp;pages=1081-1087&amp;pmid=15079193&amp;doi=10.1086/420979&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B14"><span class="mixed-citation">del Castillo I, Villamar M, Moreno-Pelayo MA. et al. A deletion involving the connexin 30 gene in nonsyndromic hearing impairment. <span><span class="ref-journal">N Engl J Med. </span>2002;<span class="ref-vol">346</span>:243â€“249.
                        doi: 10.1056/NEJMoa012052.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11807148" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1056%2FNEJMoa012052" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=A+deletion+involving+the+connexin+30+gene+in+nonsyndromic+hearing+impairment&amp;author=I+del+Castillo&amp;author=M+Villamar&amp;author=MA+Moreno-Pelayo&amp;volume=346&amp;publication_year=2002&amp;pages=243-249&amp;pmid=11807148&amp;doi=10.1056/NEJMoa012052&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B15"><span class="mixed-citation">Van Camp G, Smith RJH. Hereditary Hearing Loss Homepage. <a href="http://webhost.ua.ac.be/hhh/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://webhost.ua.ac.be/hhh/</a> Accessed 15th July, 2005.</span></li>
                    <li id="B16"><span class="mixed-citation">Guipponi M, Vuagniaux G, Wattenhofer M. et al. The transmembrane serine protease (TMPRSS3) mutated in deafness DFNB8/10 activates the epithelial sodium channel (ENaC) in vitro. <span><span class="ref-journal">Hum Mol Genet. </span>2002;
                        <span class="ref-vol">11</span>:2829â€“2836. doi: 10.1093/hmg/11.23.2829.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12393794" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1093%2Fhmg%2F11.23.2829" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+transmembrane+serine+protease+(TMPRSS3)+mutated+in+deafness+DFNB8/10+activates+the+epithelial+sodium+channel+(ENaC)+in+vitro&amp;author=M+Guipponi&amp;author=G+Vuagniaux&amp;author=M+Wattenhofer&amp;volume=11&amp;publication_year=2002&amp;pages=2829-2836&amp;pmid=12393794&amp;doi=10.1093/hmg/11.23.2829&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B17"><span class="mixed-citation">Richard I. The genetic and molecular bases of monogenic disorders affecting proteolytic systems. <span><span class="ref-journal">J Med Genet. </span>2005;<span class="ref-vol">42</span>:529â€“539. doi: 10.1136/jmg.2004.028118.</span>
                        <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1736095/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15994873" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1136%2Fjmg.2004.028118" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=The+genetic+and+molecular+bases+of+monogenic+disorders+affecting+proteolytic+systems&amp;author=I+Richard&amp;volume=42&amp;publication_year=2005&amp;pages=529-539&amp;pmid=15994873&amp;doi=10.1136/jmg.2004.028118&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B18"><span class="mixed-citation">Jovine L, Park J, Wassarman PM. Sequence similarity between stereocilin and otoancorin points to a unified mechanism for mechanotransduction in the mammalian inner ear. <span><span class="ref-journal">BMC Cell Biol. </span>2002;
                        <span class="ref-vol">3</span>:28â€“31. doi: 10.1186/1471-2121-3-28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC139993/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12445334"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [
                        <a href="//doi.org/10.1186%2F1471-2121-3-28" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Cell+Biol&amp;title=Sequence+similarity+between+stereocilin+and+otoancorin+points+to+a+unified+mechanism+for+mechanotransduction+in+the+mammalian+inner+ear&amp;author=L+Jovine&amp;author=J+Park&amp;author=PM+Wassarman&amp;volume=3&amp;publication_year=2002&amp;pages=28-31&amp;pmid=12445334&amp;doi=10.1186/1471-2121-3-28&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B19"><span class="mixed-citation">Hofmann K, Stoffel W. TMbase: A database of membrane spanning protein segments. <span><span class="ref-journal">Biol Chem. </span>1993;<span class="ref-vol">374</span>:166.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Chem&amp;title=TMbase:+A+database+of+membrane+spanning+protein+segments&amp;author=K+Hofmann&amp;author=W+Stoffel&amp;volume=374&amp;publication_year=1993&amp;pages=166&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B20"><span class="mixed-citation">Ng PC, Henikoff S. SIFT: Predicting amino acid changes that affect protein function. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3812â€“3814. doi: 10.1093/nar/gkg509.</span>
                        <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12824425" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1093%2Fnar%2Fgkg509" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=SIFT:+Predicting+amino+acid+changes+that+affect+protein+function&amp;author=PC+Ng&amp;author=S+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;doi=10.1093/nar/gkg509&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B21"><span class="mixed-citation">Fraser GR, Morgans ME, Trotter W. The syndrome of sporadic goitre and congenital deafness. <span><span class="ref-journal">Quart J Med. </span>1960;<span class="ref-vol">29</span>:279â€“295.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/13824627"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Quart+J+Med&amp;title=The+syndrome+of+sporadic+goitre+and+congenital+deafness&amp;author=GR+Fraser&amp;author=ME+Morgans&amp;author=W+Trotter&amp;volume=29&amp;publication_year=1960&amp;pages=279-295&amp;pmid=13824627&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B22"><span class="mixed-citation">Shepshelovich J, Goldstein-Magal L, Globerson A. et al. Protein synthesis inhibitors and the chemical chaperone TMAO reverse endoplasmic reticulum perturbation induced by overexpression of the iodide transporter pendrin. <span><span class="ref-journal">J Cell Sci. </span>2005;
                        <span class="ref-vol">118</span>:1577â€“1586. doi: 10.1242/jcs.02294.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15784681" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1242%2Fjcs.02294" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Sci&amp;title=Protein+synthesis+inhibitors+and+the+chemical+chaperone+TMAO+reverse+endoplasmic+reticulum+perturbation+induced+by+overexpression+of+the+iodide+transporter+pendrin&amp;author=J+Shepshelovich&amp;author=L+Goldstein-Magal&amp;author=A+Globerson&amp;volume=118&amp;publication_year=2005&amp;pages=1577-1586&amp;pmid=15784681&amp;doi=10.1242/jcs.02294&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B23"><span class="mixed-citation">Taylor JP, Metcalfe RA, Watson PF. et al. Mutations of the PDS gene. encoding pendrin, are associated with protein mislocalization and loss of iodide efflux: Implications for thyroid dysfunction in Pendred syndrome. <span><span class="ref-journal">J Clin Endocrinol Metab. </span>2002;
                        <span class="ref-vol">87</span>:1778â€“1784. doi: 10.1210/jc.87.4.1778.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11932316" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1210%2Fjc.87.4.1778" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Mutations+of+the+PDS+gene.+encoding+pendrin,+are+associated+with+protein+mislocalization+and+loss+of+iodide+efflux:+Implications+for+thyroid+dysfunction+in+Pendred+syndrome&amp;author=JP+Taylor&amp;author=RA+Metcalfe&amp;author=PF+Watson&amp;volume=87&amp;publication_year=2002&amp;pages=1778-1784&amp;pmid=11932316&amp;doi=10.1210/jc.87.4.1778&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B24"><span class="mixed-citation">Coyle B, Reardon W, Herbrick JA. et al. Molecular analysis of the PDS gene in Pendred syndrome. <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:1105â€“1112. doi:
                        10.1093/hmg/7.7.1105.
                        </span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9618167" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [
                        <a href="//doi.org/10.1093%2Fhmg%2F7.7.1105" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Molecular+analysis+of+the+PDS+gene+in+Pendred+syndrome&amp;author=B+Coyle&amp;author=W+Reardon&amp;author=JA+Herbrick&amp;volume=7&amp;publication_year=1998&amp;pages=1105-1112&amp;pmid=9618167&amp;doi=10.1093/hmg/7.7.1105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B25"><span class="mixed-citation">Li XC, Everett LA, Lalwani AK. et al. A mutation in PDS causes non-syndromic recessive deafness. <span><span class="ref-journal">Nat Genet. </span>1998;<span class="ref-vol">18</span>:215â€“217. doi: 10.1038/ng0398-215.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/9500541" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fng0398-215"
                            target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+mutation+in+PDS+causes+non-syndromic+recessive+deafness&amp;author=XC+Li&amp;author=LA+Everett&amp;author=AK+Lalwani&amp;volume=18&amp;publication_year=1998&amp;pages=215-217&amp;pmid=9500541&amp;doi=10.1038/ng0398-215&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B26"><span class="mixed-citation">Masmoudi S, Charfedine I, Hmani M. et al. Pendred syndrome: Phenotypic variability in two families carrying the same PDS missense mutation. <span><span class="ref-journal">Am J Med Genet. </span>2000;
                        <span class="ref-vol">90</span>:38â€“44. doi: 10.1002/(SICI)1096-8628(20000103)90:1&lt;38::AID-AJMG8&gt;3.0.CO;2-R.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10602116" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1002%2F(SICI)1096-8628(20000103)90%3A1%3C38%3A%3AAID-AJMG8%3E3.0.CO%3B2-R" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Pendred+syndrome:+Phenotypic+variability+in+two+families+carrying+the+same+PDS+missense+mutation&amp;author=S+Masmoudi&amp;author=I+Charfedine&amp;author=M+Hmani&amp;volume=90&amp;publication_year=2000&amp;pages=38-44&amp;pmid=10602116&amp;doi=10.1002/(SICI)1096-8628(20000103)90:1&lt;38::AID-AJMG8&gt;3.0.CO;2-R&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B27"><span class="mixed-citation">Usami S, Abe S, Weston MD. et al. Non-syndromic hearing loss associated with enlarged vestibular aqueduct is caused by PDS mutations. <span><span class="ref-journal">Hum Genet. </span>1999;<span class="ref-vol">104</span>:188â€“192.
                        doi: 10.1007/s004390050933.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10190331" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1007%2Fs004390050933" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Non-syndromic+hearing+loss+associated+with+enlarged+vestibular+aqueduct+is+caused+by+PDS+mutations&amp;author=S+Usami&amp;author=S+Abe&amp;author=MD+Weston&amp;volume=104&amp;publication_year=1999&amp;pages=188-192&amp;pmid=10190331&amp;doi=10.1007/s004390050933&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B28"><span class="mixed-citation">Van Hauwe P, Everett LA, Coucke P. et al. Two frequent missense mutations in Pendred syndrome. <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:1099â€“1104. doi: 10.1093/hmg/7.7.1099.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/9618166" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fhmg%2F7.7.1099"
                            target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Two+frequent+missense+mutations+in+Pendred+syndrome&amp;author=P+Van+Hauwe&amp;author=LA+Everett&amp;author=P+Coucke&amp;volume=7&amp;publication_year=1998&amp;pages=1099-1104&amp;pmid=9618166&amp;doi=10.1093/hmg/7.7.1099&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B29"><span class="mixed-citation">Grantham R. Amino acid difference formula to help explain protein evolution. <span><span class="ref-journal">Science. </span>1974;<span class="ref-vol">185</span>:862â€“864. doi: 10.1126/science.185.4154.862.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/4843792" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.185.4154.862"
                            target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Amino+acid+difference+formula+to+help+explain+protein+evolution&amp;author=R+Grantham&amp;volume=185&amp;publication_year=1974&amp;pages=862-864&amp;pmid=4843792&amp;doi=10.1126/science.185.4154.862&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B30"><span class="mixed-citation">Tekin M, Akcayoz D, Comak E. et al. Screening the SLC26A4 gene in probands with deafness and goiter (Pendred syndrome) ascertained from a large group of students of the schools for the deaf in Turkey. <span><span class="ref-journal">Clin Genet. </span>2003;
                        <span class="ref-vol">64</span>:371â€“374. doi: 10.1034/j.1399-0004.2003.00144.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12974744" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1034%2Fj.1399-0004.2003.00144.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Screening+the+SLC26A4+gene+in+probands+with+deafness+and+goiter+(Pendred+syndrome)+ascertained+from+a+large+group+of+students+of+the+schools+for+the+deaf+in+Turkey&amp;author=M+Tekin&amp;author=D+Akcayoz&amp;author=E+Comak&amp;volume=64&amp;publication_year=2003&amp;pages=371-374&amp;pmid=12974744&amp;doi=10.1034/j.1399-0004.2003.00144.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B31"><span class="mixed-citation">Napiontek U, Borck G, Muller-Forell W. et al. Intrafamilial variability of the deafness and goiter phenotype in Pendred syndrome caused by a T416P mutation in the SLC26A4 gene. <span><span class="ref-journal">J Clin Endocrinol Metab. </span>2004;
                        <span class="ref-vol">89</span>:5347â€“5351. doi: 10.1210/jc.2004-1013.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15531480" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]
                        [
                        <a href="//doi.org/10.1210%2Fjc.2004-1013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Intrafamilial+variability+of+the+deafness+and+goiter+phenotype+in+Pendred+syndrome+caused+by+a+T416P+mutation+in+the+SLC26A4+gene&amp;author=U+Napiontek&amp;author=G+Borck&amp;author=W+Muller-Forell&amp;volume=89&amp;publication_year=2004&amp;pages=5347-5351&amp;pmid=15531480&amp;doi=10.1210/jc.2004-1013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B32"><span class="mixed-citation">Pendred V. Deaf-mutism and goitre. <span><span class="ref-journal">Lancet. </span>1896;<span class="ref-vol">II</span>:532.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Deaf-mutism+and+goitre&amp;author=V+Pendred&amp;volume=II&amp;publication_year=1896&amp;pages=532&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B33"><span class="mixed-citation">Eyre-Walker A, Keightley PD. High genomic deleterious mutation rates in hominids. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">397</span>:344â€“347. doi: 10.1038/16915.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/9950425" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F16915"
                            target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=High+genomic+deleterious+mutation+rates+in+hominids&amp;author=A+Eyre-Walker&amp;author=PD+Keightley&amp;volume=397&amp;publication_year=1999&amp;pages=344-347&amp;pmid=9950425&amp;doi=10.1038/16915&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B34"><span class="mixed-citation">Adams J.  <span class="ref-journal">J Callow.</span> London, UK; 1815. A Philosophical Treatise on the Hereditary Peculiarities of the Human Race. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=J+Callow&amp;author=J+Adams&amp;publication_year=1815&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B35"><span class="mixed-citation">Peltonen L, Palotie A, Lange K. Use of population isolates for mapping complex traits. <span><span class="ref-journal">Nat Rev Genet. </span>2000;<span class="ref-vol">1</span>:182â€“190. doi: 10.1038/35042049.</span>
                        [<a href="https://pubmed.ncbi.nlm.nih.gov/11252747" ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2F35042049"
                            target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Use+of+population+isolates+for+mapping+complex+traits&amp;author=L+Peltonen&amp;author=A+Palotie&amp;author=K+Lange&amp;volume=1&amp;publication_year=2000&amp;pages=182-190&amp;pmid=11252747&amp;doi=10.1038/35042049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B36"><span class="mixed-citation">Ostrer H. A genetic profile of contemporary Jewish populations. <span><span class="ref-journal">Nat Rev Genet. </span>2001;<span class="ref-vol">2</span>:891â€“898.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11715044"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=A+genetic+profile+of+contemporary+Jewish+populations&amp;author=H+Ostrer&amp;volume=2&amp;publication_year=2001&amp;pages=891-898&amp;pmid=11715044&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                    <li id="B37"><span class="mixed-citation">Risch N, Tang H, Katzenstein H. et al. Geographic distribution of disease mutations in the Ashkenazi Jewish population supports genetic drift over selection. <span><span class="ref-journal">Am J Hum Genet. </span>2003;
                        <span class="ref-vol">72</span>:812â€“822. doi: 10.1086/373882.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180346/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12612865"
                            ref="reftype=pubmed&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [
                        <a href="//doi.org/10.1086%2F373882" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>]
                        <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Geographic+distribution+of+disease+mutations+in+the+Ashkenazi+Jewish+population+supports+genetic+drift+over+selection&amp;author=N+Risch&amp;author=H+Tang&amp;author=H+Katzenstein&amp;volume=72&amp;publication_year=2003&amp;pages=812-822&amp;pmid=12612865&amp;doi=10.1086/373882&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=3525152&amp;issue-id=217370&amp;journal-id=1893&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span>
                    </li>
                </ul>
            </div>
        </div>
    </div>
    <!--post-content-->

    <div id="body-link-poppers"><span/></div>
</div>